### DISCUSSION TITLE
Add translation for `name` in `directus_roles`

### DISCUSSION THREAD

**Discussion by licitdev** [directus/directus#8425] [Feature Requests]
When using Directus in a different language, the role name is always in English. 
Hence, the need to add translations for `name` in `directus_roles`.

## Before
<img width="276" alt="Screenshot 2021-09-29 at 9 13 26 PM" src="https://user-images.githubusercontent.com/26413686/135297875-8ff55256-de50-4493-a29d-bbd6e78981c2.png">

## After
<img width="276" alt="Screenshot 2021-09-29 at 9 12 52 PM" src="https://user-images.githubusercontent.com/26413686/135297892-95ef8fc8-2ac7-4724-b96d-6f613bad62ea.png">

## Trial Attempt
As role is integrated all over, it seems to be a challenge to get the name translations working for the entire app.
<img width="678" alt="Screenshot 2021-09-29 at 11 15 17 PM" src="https://user-images.githubusercontent.com/26413686/135300890-53cdfe05-4aff-4fad-a3b2-40655a843b63.png">

## Note
Hope that someone else will pick up this development. 🙈
Here's my work in progress... https://github.com/licitdev/directus/commit/b0b3caa6cb2aeef67e07ed2fad7bdf31117058c3

**Comment by rijkvanzanten** [comment]
I think this ties back into the bigger conversation around custom translations for user-generated content (labels/interface options/etc) @benhaynes

**Reply by benhaynes** [reply]
True. It will be interesting to have translation fields for some things (schema) and translation strings for others. 🤔

**Reply by rijkvanzanten** [reply]
Yeah agreed. Wondering how we can use the latter everywhere, but come up with an Input that allows for easy manipulation of the translations when you edit them 🤔 

I'm sure we can come up with a special interface that looks and feels like a text input, but then has a translations button that allows you to translate it inline, but still have it save to a centralized translations pool

**Comment by haerth** [comment]
I have the same problem. Everything is translated, only the role names are not. In case you want to give the user the chance to change the own profile data, the system shows first the role name within the navigation (wrong language in my case)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
